packages/package_info_plus/package_info_plus/example/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: package_info_example
 description: Demonstrates how to use the package_info_plus plugin.
-version: 1.2.3+4
```

Comment:
The example pubspec.yaml should not be changed

Reply:
There is a bump in `flutter_lints` package version. Yes, it shouldn't be in PR with such title, but since it is a small change I believe we can tolerate the dependency version bump. So I think it is dine that version of example itself is bumped as well.

Reply:
IMO, there's no need to bump the example project version. In this case, his breaks an integration test that should be updated.

Reply:
Ok, so I will revert the version of the example